src/elements/button/src/stories.tsx

```diff
@@ -2,77 +2,34 @@
 import * as React from 'react'
 import { css, jsx } from '@emotion/core'
 import { storiesOf } from '@storybook/react'
-import { number, text } from '@storybook/addon-knobs'
+import { boolean, number, text } from '@storybook/addon-knobs'
 import { colors } from '@uswitch/trustyle.styles'
+import { ThemeProvider } from 'theme-ui'
 
+import theme from '../../../utils/theme-selector'
 import { Icon } from '../../icon/src'
 
 import { Button } from './'
 
 const Spacer = () => <div css={css({ minHeight: 20 })} />
 
-storiesOf('Elements|Button', module).add('primary variant', () => (
+storiesOf('Elements|Button', module).addDecorator(Story => {
+  return <ThemeProvider theme={theme()}>{Story()}</ThemeProvider>
+}).add('primary variant', () => (
```

Comment:
Could this decorator be defined in `.storybook/config.js` rather than on specific component stories?

Reply:
Unfortunately not, it simply didnt work as it lost the context of the theme in the component